default.json

```diff
@@ -93,11 +93,6 @@
       "matchPackageNames": ["faker", "@types/faker"],
       "enabled": false
     },
-    {
-      "description": "Skip @faker-js/faker update to v7 (since it is ESM) [PLAT-2487]",
-      "matchPackageNames": ["@faker-js/faker"],
-      "allowedVersions": "<=6.3.1"
```

Comment:
It might be fine for our UIs but won't this be problematic for services that are not ESM?

Reply:
Identity-service is already on v7 and there doesn't seem to be an issue there: https://github.com/reside-eng/identity-service/blob/main/package.json#L71

Reply:
Okay, let's drop this then.